<?xml version="1.0"?>
<launch>
  <node pkg="hokuyo_node" type="hokuyo_node" name="hokuyo" >
    <param name="frame_id" value="base_laser" />
    <param name="min_ang" value="-1.832595714594" />
    <param name="max_ang" value="1.832595714594" />
  </node>
   <node pkg="wheel" type="MD49.py" name="wheel" />
</launch>
